# HG changeset patch
# User andrew
# Date 1374084886 -3600
#      Wed Jul 17 19:14:46 2013 +0100
# Node ID 4413a836dcc7022fa64f60d7faa491aba1248edc
# Parent  c93fcd47982ff8d85dcd2e091634b21574b4ae8a
Add note about use of $(CFLAGS)/$(CXXFLAGS)/$(CPPFLAGS) at present.

diff --git a/make/linux/makefiles/rules.make b/make/linux/makefiles/rules.make
--- openjdk/hotspot/make/linux/makefiles/rules.make
+++ openjdk/hotspot/make/linux/makefiles/rules.make
@@ -31,6 +31,9 @@
 DEMANGLE        = $(DEMANGLER) < $@ > .$@ && mv -f .$@ $@
 
 # $(CC) is the c compiler (cc/gcc), $(CXX) is the c++ compiler (CC/g++).
+# FIXME: $(CXXFLAGS) currently only includes preprocessor flags while
+# $(CFLAGS) includes C and C++ flags.  Ideally, there should be three
+# variables: $(CFLAGS), $(CXXFLAGS) and $(CPPFLAGS).
 CC_COMPILE       = $(CC) $(CXXFLAGS)
 CXX_COMPILE      = $(CXX) $(CXXFLAGS) $(CFLAGS)
 
